### PR TITLE
Clarify Arguments/Parameters in index.md

### PR DESCRIPTION
**Pull request by cc-erty**
Clarification of the difference between **arguments** and **parameters**.

http://stackoverflow.com/questions/1788923/parameter-vs-argument

### CODE REVIEW

**Comment by tkellen** [comment]
cc @bobholt or @leobalter

File: content/javascript-basics/index.md [hunk 1]
```diff
@@ -111,15 +111,15 @@ Functions are a fundamental building block of JavaScript programs; they provide
...
+This function takes two **parameters**, `a` and `b`. It adds them together, and returns the result.
 
 This is a valid bit of JavaScript, but as written, there's no way for us to *call* our function if we actually want to add two things. We can solve this by assigning our function to a variable:
 
     var addTwoNumbers = function(a, b) {
       return a + b;
     };
 
-What we've done here is take a **function expression** and assign it to a variable. Now that we've done this, we can call our function by using the name of the variable:
+What we've done here is take a **function expression** and assign it to a variable. Now that we've done this, we can call our function by using the name of the variable, passing `1` and `2` as **arguments** into the function:
```
Comment (bobholt) [inline content/javascript-basics/index.md:122]: > passing `1` and `2`

The example below passes `1` and `1` as arguments - either this statement or the example should be changed to match.

**Comment by bobholt** [comment]
Thanks for the contribution - just 1 comment.

File: content/javascript-basics/index.md (hunk 1 above)
Comment (cc-erty) [inline content/javascript-basics/index.md:122]: Correct, I misread the example. I've updated the code.

**Comment by bobholt** [comment]
Thanks, @cc-erty!

**Pull request merged**